### PR TITLE
inveniordm v8 update

### PR DESCRIPTION
**Pull request by fenekku**
- modernize code base
- release: 2022.05.06

### CODE REVIEW

**Comment by fenekku** [comment]
Hmm GitHub is acting strangely... Perhaps it's the large lcsh vocabulary file...? Checkout the branch locally and compare against master to see the difference :shrug:

**Comment by Meowcenary** [comment]
I thought I could get to this today, but had some stuff come up. I'll check this out on Monday if it's not already merged.

**Review by carsonicator — APPROVED** [review]
Thanks @fenekku

File: invenio_subjects_lcsh/writer.py [hunk 1]
```diff
@@ -1,27 +1,33 @@
...
-    """
-    with open(filepath, "w") as f:
-        yaml.dump(list(entries), f)
+    def yaml(self, filepath):
+        """Write out to yaml.
 
-    return filepath
+        We don't rely on the yaml library to write out the values since it
+        can't stream-write. Because the values are simple, this is a
+        reasonable choice.
```
Comment (sharpattack) [inline invenio_subjects_lcsh/writer.py:27]: 👍

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -5,9 +5,6 @@ on:
     branches: master
   pull_request:
     branches: master
-  schedule:
-    # * is a special character in YAML so you have to quote this string
-    - cron:  '0 3 * * 6'
```
Comment (sharpattack) [inline .github/workflows/tests.yml:10]: Turned out to be overkill?

**Review by sharpattack — APPROVED** [review]
Looks good to me. I was not able to see the diff for the second commit, but based off the commit message that it's just the vocabulary files should be fine.

**Pull request merged**